setup.cfg

```diff
@@ -26,6 +26,7 @@ packages = find:
 install_requires =
     black
     typing-extensions ; python_version < "3.8"
+    dataclasses ; python_version < "3.7"
 python_requires = >=3.6
```

Comment:
You can also considered being 3.7+ only ...

Reply:
Well, Black and isort both still support Python 3.6, and maybe it makes sense to be in sync with them. But it's a good point, we definitely should drop support for old Python versions periodically.